index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li><img src="My Logo.svg" alt="Logo svg" /></li>
+        <li><img src="Union.png" alt="Menu Button" /></li>
+      </ul>
```

Comment:
- [ ] Kindly make sure this logo is actually a link. You may surround the logo with an `<a>` tag. 👌

> This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- [ ] I suggest wrap The hamburger/menu button with a `nav` tag 💯
> The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

For example, your code should be something like this 💡
```
<header>
  <a href="#"><img src="My Logo.svg" alt="Logo svg" /></a>
     <nav>
         <img src="Union.png" alt="Menu Button" />
     </nav>
</header>
```